src/GovUk.Education.ExploreEducationStatistics.Content.Services/Interfaces/IReleaseService.cs

```diff
@@ -1,23 +1,20 @@
 ﻿#nullable enable
+using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using GovUk.Education.ExploreEducationStatistics.Common.Model;
 using GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels;
-using GovUk.Education.ExploreEducationStatistics.Publisher.Model.ViewModels;
 using Microsoft.AspNetCore.Mvc;
 
 namespace GovUk.Education.ExploreEducationStatistics.Content.Services.Interfaces
 {
     public interface IReleaseService
     {
-        Task<Either<ActionResult, ReleaseViewModel>> GetCachedViewModel(string publicationSlug, string? releaseSlug = null);
+        Task<Either<ActionResult, CachedReleaseViewModel>> GetRelease(string publicationSlug,
+            string? releaseSlug);
 
-        Task<Either<ActionResult, ReleaseSummaryViewModel>> GetSummary(string publicationSlug, string? releaseSlug = null);
+        Task<Either<ActionResult, CachedReleaseViewModel>> GetRelease(Guid releaseId, DateTime? expectedPublishDate);
```

Comment:
CachedReleaseViewModel` was a class that already existed but when it's produced here its not cached. I see your confusion now 🙂 .

The service that produced it originally was in the Publisher. See `Publisher.Services.Inferfaces.IReleaseService` on the dev branch.

```
Task<CachedReleaseViewModel> GetLatestReleaseViewModel(Guid publicationId,
  IEnumerable<Guid> includedReleaseIds,
  PublishContext context);

Task<CachedReleaseViewModel> GetReleaseViewModel(Guid id, PublishContext context);
```

The Content API retrieved this so in `Content.Services.Interfaces.IReleaseService` there was a method `GetCachedRelease`

```
Task<Either<ActionResult, CachedReleaseViewModel?>> GetCachedRelease(string publicationSlug, string? releaseSlug = null);
```

This cached resource was used by a `GetCachedViewModel` in the same service where it was transformed and the publication and its methodologies are added all in the constructor of `ReleaseViewModel`.

```
Task<Either<ActionResult, ReleaseViewModel>> GetCachedViewModel(string publicationSlug, string? releaseSlug = null);
````

In this PR I didn't want to change what's cached or do too much renaming so the underlying service gets the same `CachedReleaseViewModel` as it did before in Publisher. That's now in `Content.Services` where you've commented on it here.

The new `IReleaseCacheService` offers the following method which wraps that, getting it if it's not already cached. A couple of other places use this where they want the cached release on it's own.

```
Task<Either<ActionResult, CachedReleaseViewModel>> GetRelease(string publicationSlug, string? releaseSlug = null);
```

There's then a `GetReleaseAndPublication` in the same `IReleaseCacheService` which is the equivalent of the old `ReleaseService.GetCachedViewModel`.

```Task<Either<ActionResult, ReleaseViewModel>> GetReleaseAndPublication(string publicationSlug, string? releaseSlug = null);
```

I wasn't sure about the name `CachedReleaseViewModel` and wanted to try and eliminate it which is why I raised EES-3719. I also don't think `IReleaseCacheService.GetReleaseAndPublication` should exist, or at least not here because it means that service depends on `IMethodologyCacheService` and `IPublicationCacheService`. I'm attempting to resolve these issues in EES-3719 which is https://github.com/dfe-analytical-services/explore-education-statistics/pull/3509

Reply:
Cool, yeah I'd probably try to make the name disappear some point down the line, but it's fine here.